### PR TITLE
fix #172

### LINKED ISSUE #172: Destroying a recording should revert it to a sound-less cell
Only when done by the user though. Programmatically destroying should still just destroy the whole thing

### PR DESCRIPTION
**Pull request by mahdoosh1**
i fixed #172 but not sure of `[]` when returning

### CODE REVIEW

**Comment by TodePond** [comment]
Hey thanks for taking a look at this! :) It looks like there are a couple of issues here which I'm happy to help with, as some of my code is quite messy. Are you able to run your branch on your machine to test it out? We can take it from there!

**Comment by mahdiyadiilavi1** [comment]
i dont have a machine sorry. cuz i have a phone and did coding on this.

**Comment by mahdiyadiilavi1** [comment]
if you are busy, im doing it rn

**Comment by mahdiyadiilavi1** [comment]
ok looks like i cant install deno on my mobile with linux terminal because it uses aarch64

**Comment by TodePond** [comment]
oh hey you don't need to use deno, that's just one way of running a file server :) I'm not sure what the best way to run a local file server on mobile, but maybe there's a way!

**Comment by mahdiyadiilavi1** [comment]
i see it

**Comment by mahdiyadiilavi1** [comment]
how do i get deletion cell?

**Comment by TodePond** [comment]
Awesome!


> how do i get deletion cell?

Currently you unlock it like this:

1. Create two cells.
2. Create three wires.

We need to add cheat codes for testing though. (https://github.com/TodePond/Arroost/issues/118)

**Comment by mahdiyadiilavi1** [comment]
i have a browser that has console, not sure how to enable debugging

**Comment by mahdiyadiilavi1** [comment]
i am very sorry. looks like i am not able to test it since my browser isnt working.

i want you to test this youself because it is easier and faster on your machine than my mobile device

**Comment by mahdiyadiilavi1** [comment]
Ah. didnt work

**Comment by mahdiyadiilavi1** [comment]
Luke, i dont know why it isnt working.

But it seems problem is from setting or getting hasSound variable.

I am not familiar with structure of Entity.
So please do it youself.

File: source/arroost/entities/cells/destruction.js [hunk 1]
```diff
@@ -152,10 +154,15 @@ export class Destruction extends Entity {
 							break
 						}
 					}
+					const operations = archiveCell(shared.nogan, id)
+					operations.push(...wireOperations)
+					return operations
+				}
+				if (cell.hasSound) {
+					cell.hasSound.set(False)
```
Comment (TodePond) [inline source/arroost/entities/cells/destruction.js:162]: hey looks like this should be `false` instead of `False` :)

**Pull request closed without merging**